### ISSUE TITLE
Problem with version option

### ISSUE THREAD

**Issue by nfroidure** [tj/commander.js#558]
When creating a program and giving it a version, you are not able anymore to pass the `version` raw arg witch is pretty annoying and unexpected.

I encountered this issue with [this project](https://github.com/SimpliField/miniquery). It allows querying JSON files for its contents.

``` sh
# Getting the name in a package.json works
cat package.json | bin/miniquery.js name
#> miniquery

# Getting the version in a package.json will always return the miniquery program version
cat package.json | bin/miniquery.js version
#> 1.1.0
```

I assume the error would also exist with trying to open a file named version in a CLI tool made with commander.

I forked commander and the problem seems identical for the current master. I think the problem sits in this implicit thing: https://github.com/tj/commander.js/blob/master/index.js#L196-L198 Murphy's law FTW 🎲

I think version should only be printed for an explicit -v or --version arg or at least, the current behaviour should be configurable to avoid this issue.

For ref. the actual CLI tool code:
https://github.com/SimpliField/miniquery/blob/master/bin/miniquery.js#L9

**Comment by nfroidure** [comment]
FYI the temp workaround i had to do https://github.com/SimpliField/miniquery/pull/2/files

**Comment by shadowspawn** [comment]
This issue has not had any activity in over six months. It isn't likely to get acted on due to this report.

Feel free to open a new issue if it comes up again, with new information and renewed interest.
 
Thank you  for your contributions.

**Comment by shadowspawn** [comment]
Possible fixed by #494
